utilities/blob_db/blob_file.h

```diff
@@ -116,6 +124,24 @@ class BlobFile {
   // once the file is created, this never changes
   uint64_t BlobFileNumber() const { return file_number_; }
 
+  // Get the set of SST files whose oldest blob file reference points to
+  // this file.
+  const SstFileSet& GetParentSstFiles() const { return parent_sst_files_; }
+
+  // Add an SST file whose oldest blob file reference points to this file.
+  void AddParentSstFile(uint64_t sst_file_number) {
+    auto it = parent_sst_files_.find(sst_file_number);
+    assert(it == parent_sst_files_.end());
+    parent_sst_files_.insert(it, sst_file_number);
```

Comment:
I'm not very expert on this but does a parent_sst_files_.end() hint useful to unordered_set at all? If we are not sure, maybe we don't have to use it so that it doesn't confuse people like me:)